src/clj/harja/palvelin/palvelut/kulut/valikatselmukset.clj

```diff
@@ -0,0 +1,94 @@
+(ns harja.palvelin.palvelut.kulut.valikatselmukset
+  (:require
+    [com.stuartsierra.component :as component]
+    [slingshot.slingshot :refer [throw+ try+]]
+    [harja.palvelin.komponentit.http-palvelin :refer [julkaise-palvelu poista-palvelut]]
+    [harja.kyselyt.urakat :as q-urakat]
+    [harja.kyselyt.valikatselmus :as q]
+    [harja.domain.kulut.valikatselmus :as valikatselmus]
+    [harja.domain.urakka :as urakka]
+    [harja.domain.muokkaustiedot :as muokkaustiedot]
+    [harja.pvm :as pvm]
```

Comment:
Itse tykkään sellaisesta ajatuksesta, että nämä mukaan liitettävät komponentit olisivat aakkos/loogisuus tms. järjestyksessä eli tyyliin:
harja.domain alkuiset olisivat peräkkäin.

Reply:
Järjestetty aakkosjärjestykseen, poikkeuksena ei-harja-namespacet, jotka laitoin ensimmäisiksi.